### PR TITLE
Task/sdk 3646/support ct v6.0.0

### PR DESCRIPTION
**Pull request by Anush-Shand**
Adds support for CT SDK v6.0.0 (thereby also v5.2.2)
(release for supporting ct-sdk v5.2.2 was skipped)

### CODE REVIEW

**Comment by Anush-Shand** [comment]
Closing this PR as changes already merged into iOS branch https://github.com/CleverTap/clevertap-react-native/pull/367

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,34 @@
...
 ==========
 
+Version 2.0.0 *(15 February 2024)*
+-------------------------------------------
+**What's new**
+* **[Android Platform]**
+  * Supports [CleverTap Android SDK v6.0.0](https://github.com/CleverTap/clevertap-android-sdk/blob/master/docs/CTCORECHANGELOG.md#version-600-january-15-2024).
+
+
+* **[iOS Platform]**
```
Comment (shivamsharma2710) [inline CHANGELOG.md:11]: I believe the iOS PR has resolved the changeLog todos.

**Review by shivamsharma2710 — APPROVED** [review]
Looks good other than changeLog todos.

File: CHANGELOG.md (hunk 1 above)
Comment (Anush-Shand) [reply CHANGELOG.md:11]: Yess. They have been resolved

**Pull request merged**